CHANGES

```diff
@@ -6,13 +6,19 @@ are in form of *headline.major.minor* numbers. Backwards-compatible changes
 increment the minor version number only.
 
 
-Version 2.2.1
+Version 2.3.0
 -------------
 
 Released PENDING
 
-* Add support for loading configuration file in ``nameko shell`` via ```--config`` option
-* Changed :class:`~nameko.web.handlers.HttpRequestHandler` to allow override how web exceptions are handled
+* Add support for loading configuration file in ``nameko shell`` via ``--
+  config`` option
+* Changed :class:`~nameko.web.handlers.HttpRequestHandler` to allow override
+  how web exceptions are handled
+* Enabled reliable delivery on broadcast events when combined with a custom
+  :py:attr:`~nameko.events.EventHandler.broadcast_identity`. Reliable delivery
+  now defaults to enabled for all handler types. It must be explicitly turned
+  off with broadcast mode unless override the default `broadcast_identity`.
 
```

Comment:
since you are bumping the version here, i think you should also add a changelog entry about the bumped minimum eventlet requirements (due to eventlet removing them from pypi)